source/crud/crud.rst

```diff
@@ -661,9 +661,27 @@ older sharded clusters.
 estimatedDocumentCount
 ~~~~~~~~~~~~~~~~~~~~~~
 
-The estimatedDocumentCount function is implemented using the `count` command
-with no query filter, skip, limit, or other options that would alter the
-results. As documented above, the only supported option is maxTimeMS.
+On server versions greater than or equal to 4.9.0 (wire version 12 or higher),
+the estimatedDocumentCount function is implemented using the ``$collStats``
+aggregate pipeline stage with ``$group``. As documented above, the only supported
+option is maxTimeMS::
```

Comment:
> I wrote 5.0, but why wouldn't it be 4.9?

I suggested that for consistency with existing entries, since the table only refers to stable server releases.

I realize that differs from how we refer to pre-release versions in `minServerVersion` test requirements (e.g. "4.1.8" for some sharded transaction tests).